python/taichi/ui/scene.py

```diff
@@ -93,8 +93,18 @@ def lines(self,
               width,
               indices=None,
               color=(0.5, 0.5, 0.5),
-              per_vertex_color=None):
+              per_vertex_color=None,
+              vertex_offset: int = 0,
+              vertex_count: int = None,
+              index_offset: int = 0,
```

Comment:
ditto

Reply:
The default value is 0 which is a default behavior for the user